src/DB.php

```diff
@@ -758,6 +766,7 @@ public static function insertMessageRequest(Message &$message)
             $sth->bindParam(':message_id', $message_id, \PDO::PARAM_INT);
             $sth->bindParam(':user_id', $from_id, \PDO::PARAM_INT);
             $sth->bindParam(':date', $date, \PDO::PARAM_STR);
+            $sth->bindParam(':edit_date', $edit_date, \PDO::PARAM_STR);
```

Comment:
Mind posting a super-simple command that uses the callback query you're talking about please?
Haven't looked into those that much yet and it would make my life a lot easier 😊 
Then I'll understand exactly what you mean and we can find a nice solution!